lib/tool_shed/util/shed_util_common.py

```diff
@@ -512,6 +512,11 @@ def get_repository_for_dependency_relationship( app, tool_shed, name, owner, cha
                                                owner=owner,
                                                changeset_revision=changeset_revision )
     if not repository:
+        tool_shed_url = common_util.get_tool_shed_url_from_tool_shed_registry( app, tool_shed )
+        repository_clone_url = os.path.join( tool_shed_url, 'repos', owner, name )
+        repo_info_tuple = (None, repository_clone_url, changeset_revision, None, owner, None, None)
+        repository, pcr = repository_was_previously_installed( app, tool_shed_url, name, repo_info_tuple )
```

Comment:
@gregvonkuster It seems that the code at [line 503-507](https://github.com/gregvonkuster/galaxy/blob/a16cb06705676aa49e725c8ab58521c239dfea4d/lib/tool_shed/util/shed_util_common.py#L503-L507) is a subset of what is done by `repository_was_previously_installed()`: can this be consolidated in this last function?